subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/screenshot/ScreenshotCapturerImpl.kt

```diff
@@ -20,10 +19,7 @@ class ScreenshotCapturerImpl(private val testArtifactsProvider: TestArtifactsPro
     // todo remove in avito and then here
     @Suppress("unused")
     constructor(outputDirectory: Lazy<File>) : this(
-        testArtifactsProvider = UniqueDirTestArtifactsProvider(
-            rootDir = outputDirectory,
-            testDirGenerator = TestDirGenerator.Stub
-        )
+        testArtifactsProvider = TestArtifactsProviderFactory.createStub(outputDirectory)
```

Comment:
I could misunderstand the `createStub` name in the future

Reply:
Because it's a production code and I don't know what kind of Stub we create

Reply:
It's a temporary  solution for backward compatibility
Take a look at: https://github.com/avito-tech/avito-android/blob/854f5a03b6c9fede37fc41d85ef4f246f79a8408/subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/screenshot/ScreenshotCapturerImpl.kt#L22

It's only usage and will be removed in next release